girder/web_client/src/views/widgets/HierarchyWidget.js

```diff
@@ -224,13 +268,34 @@ var HierarchyWidget = View.extend({
                 showSizes: this._showSizes,
                 selectedItem: this._defaultSelectedResource,
                 highlightItem: this._highlightItem,
+                paginated: this._paginated,
                 parentView: this
             });
             this.listenTo(this.itemListView, 'g:itemClicked', this._onItemClick);
             this.listenTo(this.itemListView, 'g:checkboxesChanged', this.updateChecked);
+
             this.listenTo(this.itemListView, 'g:changed', () => {
                 this.itemCount = this.itemListView.collection.length;
                 this._childCountCheck();
+                if (this._paginated && this.hierarchyPaginated && this.itemListView.getNumPages() > 1) {
+                    this.render();
+                    this.$('.g-hierarchy-breadcrumb-bar').addClass('g-hierarchy-sticky');
```

Comment:
Do you think we should always have the breadcrumbs sticky?

Reply:
It's a bigger change to the UI and I wasn't sure if in all uses of hierarchyWidget it made sense.  It feels like something that should be done but with some other considerations, like adding the action bar (folder/item actions) to be under the same root element, that way the breadcrumb and action bar are both static at the same time.  As of right now if used outside of browserwidget the actionbar is treated as another scrolling element.